src/Extension/Task/GenerateAppTask.php

```diff
@@ -65,7 +66,7 @@ public function getPlaceholders(): array
                 [
                     'name' => 'app-name',
                     'description' => 'Input name for new App',
-                    'type' => 'string',
+                    'type' => ValueTypeEnum::TYPE_STRING,
```

Comment:
Task and task set versions must be increased respectively. The same for the rest of php commands